src/components/Dashboard/TodoList.vue

```diff
@@ -615,60 +610,60 @@ export default class TodoList extends Mixins(
   @Action setHasBlockerTask!: ActionBindingIF
 
   /** The Auth Web URL string. */
-  private get authWebUrl (): string {
+  get authWebUrl (): string {
     return sessionStorage.getItem('AUTH_WEB_URL')
   }
 
   /** The Edit URL string. */
-  private get editUrl (): string {
+  get editUrl (): string {
     return sessionStorage.getItem('EDIT_URL')
   }
 
   /** The Create URL string. */
-  private get createUrl (): string {
+  get createUrl (): string {
     return sessionStorage.getItem('CREATE_URL')
   }
 
   /** The Manage Businesses URL string. */
-  private get manageBusinessesUrl (): string {
+  get manageBusinessesUrl (): string {
     return sessionStorage.getItem('AUTH_WEB_URL') + 'business'
   }
 
   /** The BCROS Home URL string. */
-  private get bcrosHomeUrl (): string {
+  get bcrosHomeUrl (): string {
     return sessionStorage.getItem('BUSINESSES_URL')
   }
 
   /** The Base URL string. */
-  private get baseUrl (): string {
+  get baseUrl (): string {
     return sessionStorage.getItem('BASE_URL')
   }
 
   /** The Business ID string. */
-  private get businessId (): string {
+  get businessId (): string {
     return sessionStorage.getItem('BUSINESS_ID')
   }
 
   /** The Incorporation Application's Temporary Registration Number string. */
-  private get tempRegNumber (): string {
+  get tempRegNumber (): string {
     return sessionStorage.getItem('TEMP_REG_NUMBER')
   }
 
   /** Whether filing an Alteration is required (ie, there is a Todo filing). */
-  private get requiresAlteration (): boolean {
+  get requiresAlteration (): boolean {
     if (this.isBcCompany || this.isUlc) {
       return this.getTasks.some(task => task.task?.filing?.header?.name === FilingTypes.ALTERATION)
     }
     return false
   }
 
   /** Alteration action button label. */
-  private get alterationBtnLabel (): string {
+  get alterationBtnLabel (): string {
     return this.requiresAlteration ? 'Alter Now' : 'Resume'
   }
 
   /** Whether the File Annual Report button should be disabled. */
-  private isFileAnnualReportDisabled (item: TodoItemIF, index: number): boolean {
+  protected isFileAnnualReportDisabled (item: TodoItemIF, index: number): boolean {
```

Comment:
This is called from the template so it needs to be protected otherwise my VS Code plugins tell me this is not used.